File: Week 1/03. Mario (more)/mario.c
// Print mario pyramid
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Prompt user for valid height
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print pyramid
    for (int i = 0; i < height; i++)
    {
        // Left side of pyramid
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("  ");
        // Right side of pyramid
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6/03. Sentimental Mario (more)/mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break

    for i in range(height):
        print(" " * (height - i - 1) + "#" * (i + 1) + "  " + "#" * (i + 1))

if __name__ == "__main__":
    main()
